setup.py

```diff
@@ -5,10 +5,15 @@
 with open(join(dirname(abspath(__file__)), 'requirements.txt')) as f:
     requirements = f.readlines()
 
+with open("readme.md", "r") as fh:
```

Comment:
This seems to be `README.md` for this package.

Reply:
:facepalm: thanks...